k8s/service.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: palworld-server
+  name: palworld-server
+  namespace: default
+spec:
+  externalTrafficPolicy: Cluster
+  internalTrafficPolicy: Cluster
+  ipFamilies:
+  - IPv4
+  ports:
+  - name: palworld1
+    port: 8211
+    protocol: UDP
+    targetPort: 8211
```

Comment:
targetPort should be `8211-palworld`

Reply:
`targetPort` isn't needed if `port` is the same, correct?

Reply:
targetport is what is being mapped (service will expose 8211), it's good to point to the port 'name' that's on the deployment (since it was named 8211-palworld)